package.json

```diff
@@ -4,26 +4,30 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "cross-env NODE_ENV=development nyc mocha --require @babel/register ./src/__tests__/*.js",
+    "test": "cross-env NODE_ENV=development nyc mocha --require @babel/register ./src/__tests__/*.js --timeout 45000 --exit",
     "coverage": "nyc report --reporter=text",
     "coveralls": "nyc report --reporter=text-lcov | coveralls",
     "lint": "eslint \"**/*.js\" --ignore-pattern node_modules/ --fix",
     "build": "babel ./src -d dist",
     "start": "node dist/index.js",
-    "start:dev": "nodemon --exec babel-node -- src/index.js",
+    "sync": "DEBUG=dev babel-node ./src/database/reSync",
```

Comment:
what's the purpose of the `sync` script?

Reply:
This handles the creation of tables and seeding for testing/development environment.

Reply:
ok